boards/FSM/Inc/App/configs/App_WheelSpeedThresholds.h

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+#define LEFT_WHEEL_MIN_SPEED 0.10772f
```

Comment:
A comment here explaining why 0.10772f is chosen instead of 0 would be nice

Reply:
done